### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by aleks-pro**
/cc @churkin @LavrovArtem

### CODE REVIEW

File: test/client/fixtures/sandbox/event/unload-test.js [hunk 1]
```diff
@@ -30,6 +30,17 @@ asyncTest('BEFORE_UNLOAD_EVENT must be called last (GH-400)', function () {
         });
 });
 
+test('Hammerhead UNLOAD_EVENT must be called (GH-1095)', function () {
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/event/unload-test.js]: Why do you add `(GH-1095)` suffix to the test name?
test name should be in lowercase.

File: test/client/fixtures/sandbox/event/unload-test.js [hunk 2]
```diff
@@ -30,6 +30,17 @@ asyncTest('BEFORE_UNLOAD_EVENT must be called last (GH-400)', function () {
...
+    var unloadSandbox   = hammerhead.sandbox.event.unload;
+    var unloadWasCalled = false;
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/event/unload-test.js]: `var unloadEventWasCalled = `

File: test/client/fixtures/sandbox/event/unload-test.js [hunk 3]
```diff
@@ -30,6 +30,17 @@ asyncTest('BEFORE_UNLOAD_EVENT must be called last (GH-400)', function () {
...
+
+    hammerhead.on(hammerhead.EVENTS.unload, () => {
+        unloadWasCalled = true;
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/event/unload-test.js]: Don't use arrow functions in tests.

Need to unsubscribe a test handler
```js
var handler = function () {
      unloadWasCalled = true;
       hammerhead.off(hammerhead.EVENTS.unload, handler);
};
 hammerhead.on(hammerhead.EVENTS.unload, handler);
```

File: test/client/fixtures/sandbox/event/unload-test.js (hunk 1 above)
Comment (aleks-pro) [reply test/client/fixtures/sandbox/event/unload-test.js]: I thought, if you fix an issue and write a test for this case, you need to add such a suffix with issue number. Am i wrong?

File: test/client/fixtures/sandbox/event/unload-test.js [hunk 4]
```diff
@@ -30,6 +30,19 @@ asyncTest('BEFORE_UNLOAD_EVENT must be called last (GH-400)', function () {
         });
 });
 
+test('hammerhead UNLOAD_EVENT must be called', function () {
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/event/unload-test.js]: We don't use a `must` work in test names (from recent time ).
So, please, rename your test to `UNLOAD_EVENT should be called`.

Also, we have a small piece of unstable tests in hammerhead and testcafe.
You PR don't affect tests.
To restart tests just add a comment `@testcafe-build-bot  \retest`

**Comment by aleks-pro** [comment]
@testcafe-build-bot \retest

**Comment by aleks-pro** [comment]
FPR

**Pull request merged**